homeassistant/components/rachio/__init__.py

```diff
@@ -30,35 +22,12 @@
 
 SUPPORTED_DOMAINS = ["switch", "binary_sensor"]
 
-CONFIG_SCHEMA = vol.Schema(
```

Comment:
We should keep `CONFIG_SCHEMA`, but mark it deprecated.

```py
CONFIG_SCHEMA = cv.deprecated(DOMAIN)
```

This gives the user at least a log/warning that their YAML entry isn't used.

Reply:
Done